seshdash/tasks.py

```diff
@@ -97,7 +97,6 @@ def generate_auto_rules(site_id):
 
 @shared_task
 def get_BOM_data():
-    print "Getting started getting the bom point "
```

Comment:
this bit was a removed. Can you pull the master first?
` logger.debug("got raw date %s with timezone %s"%(                
     sys_data['VE.Bus state']['timestamp'],                       
     site.time_zone                                               
     ))                                                           
 date = time_utils.epoch_to_datetime(float(sys_data['VE.Bus       
 tz=site.time_zone)                                               
 logger.debug("saving before localize  BOM data point with time   
`
what it looks like